docs/clickhouse.rst

```diff
@@ -32,6 +32,12 @@ For macos follow instructions at https://clickhouse.com/docs/en/getting-started/
 
 If you already have clickhouse installed, please symlink it to ``/usr/bin/clickhouse``
 
+Customize ClickHouse start timeout
```

Comment:
Please add section Environment variables, and describe each variable under corresponding title, e.g.:

```
Environment variables
---------------------------------

TESTSUITE_CLICKHOUSE_SERVER_START_TIMEOUT
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

...

```